ansible/roles/test/tasks/crm/crm_test_acl_counter.yml

```diff
@@ -0,0 +1,62 @@
+- block:
+
+    - set_fact: crm_stats_acl_counter_used=0
+    - set_fact: crm_stats_acl_counter_available=0
+
+    - name: Copy ACL JSON config to switch.
+      copy: src=roles/test/tasks/crm/acl.json dest=/tmp
+
+    - name: Add ACL
+      command: acl-loader update full /tmp/acl.json
+
+    - name: Get ACL table key
+      command: bash -c "docker exec -i database redis-cli --raw -n 2 KEYS *CRM:ACL_TABLE_STATS* | head -1"
```

Comment:
This is fetching the first table entry and checking for values. But if there are multiple tables, and if the acl-loader is updating entries to a table other than the first entry, then this test fails.